src/UmamiApiClient.ts

```diff
@@ -49,7 +49,7 @@ export class UmamiApiClient {
 
     log(`GET ${dest}`);
 
-    return get(dest, undefined, this.getHeaders(headers));
+    return get(dest, undefined, { ...this.getHeaders(headers), ...{ 'Content-Type': undefined } });
```

Comment:
Why do we need to override content type here? The default is `application/json`.

Reply:
In my testing, keeping the `Content-Type` header on get requests to the API resulted in error 500.

But here in the `next-basics` package you can see it's being included by default https://github.com/umami-software/next-basics/blob/master/src/client/request.ts#L14C17-L14C17

Removing that override causes it to just error 500 unfortunately. The tests should successfully complete for `await apiClient.client.getWebsiteStats` but they don't without disabling that header.

Admittedly I haven't re-tested since the new update that provides better information from the API and I couldn't replicate the bug with a manually written fetch request so I assume that header is the sole problem here.

Reply:
@mikecao I just retested without the header and it still results in just an error 500 with the latest API version.